docker-entrypoint.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+function stop {
+	kill $(ps -C ts3server -o pid= | awk '{ print $1; }')
+	exit
+}
+
+trap stop INT
+trap stop TERM
+
+test -d /data/files || mkdir -p /data/files && chown teamspeak:teamspeak /data/files
+ln -sf /data/files $TS_DIRECTORY/files
+
+test -d /data/logs || mkdir -p /data/logs && chown teamspeak:teamspeak /data/logs
+ln -sf /data/logs $TS_DIRECTORY/logs
+
+ln -sf /data/query_ip_whitelist.txt $TS_DIRECTORY/query_ip_whitelist.txt
```

Comment:
I have an idea of how to make this more robust.  Instead of specifying a static list of files, it might be better to do something like:
```
touch /data/query_ip_whitelist.txt /data/query_ip_blacklist.txt /data/ts3server.ini /data/ts3server.sqlitedb /data/ts3server.sqlitedb-shm /data/ts3server.sqlitedb-wal
for i in $(ls /data)
do
  ln -sf /data/${i} $TS_DIRECTORY/${i}
done
```

This would create all of the files necessary and then if there are additional files, either from new features or other configuration setup by the user, it would automatically symlink them.